packages/page-staking/src/Actions/Account/ListNominees.tsx

```diff
@@ -42,7 +48,10 @@ function ListNominees ({ nominating, stashId }: Props): React.ReactElement<Props
         </Expander>
       )}
       {nomsChilled && nomsChilled.length !== 0 && (
-        <Expander summary={t<string>('Renomination required ({{count}})', { replace: { count: nomsChilled.length } })}>
+        <Expander
+          help={t<string>('The validators that will never be selected again by the Phragmen algorithm. Either because they stopped validating, or because they got slashed while you were one of their nominator. If you wish to nominate them again, you need to manually select them again.')}
```

Comment:
This is auto-chilled. So they were slashed.

Reply:
```suggestion
          help={t<string>('The validators that got slashed and for which your nomination got removed. If you wish to nominate them again, you need to manually select them again.')}
```